be/src/vec/runtime/vdatetime_value.h

```diff
@@ -157,7 +157,30 @@ class VecDateTimeValue { // Now this type is a temp solution with little changes
               _month(0), // so this is a difference between Vectorization mode and Rowbatch mode with DateTimeValue;
               _year(0) {} // before int128  16 bytes  --->  after int64 8 bytes
 
-    explicit VecDateTimeValue(int64_t t) { from_date_int64(t); }
+    static VecDateTimeValue create_from_olap_date(uint64_t value) {
```

Comment:
add comment what's the different between `create_from_olap_datetime` and `create_from_normal_datetime`. it is default to know the different from the name of function name.